test/jit/test_types.py

```diff
@@ -21,6 +21,22 @@
                        "instead.")
 
 class TestTypesAndAnnotation(JitTestCase):
+    def test_pep585_type(self):
+        def fn(x: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+            xl: list[torch.Tensor] = []
+            xd: dict[str, torch.Tensor] = {}
```

Comment:
Nit:
`List[toch.Tensor]` and `Dict[str, torch.Tensor]` happens to be default types for `[]` and `{}`, so the test case doesn't exact test that PEP 585 is correctly parsed. Do you mind trying some other types like `List[int]`, Dict[str, int]`? Thanks